android/libraries/rib-android/src/main/kotlin/com/uber/rib/core/XRayConfig.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2023. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.uber.rib.core
+
+/**
+ * Configuration for XRay.
+ *
+ * @property enabled `true` to enable XRay. By default it is disabled.
+ * @property alphaEnabled `true` to enable alpha changes when XRay is enabled.
+ */
+public data class XRayConfig(
+  val enabled: Boolean = false,
+  val alphaEnabled: Boolean = true,
+)
```

Comment:
This is so simple IMO it should live on `XRay.kt` file at the top level. No need for a new file

Reply:
Done